client/src/HowItWorks.tsx

```diff
@@ -1,7 +1,341 @@
+                        Still have questions?
+                    </Typography>
+                    <Box className={classes.questions}>
+                        <QuestionList questionList={tabSelected === "nonprofit" ? nonprofitQuestions : citizenQuestions}></QuestionList>
+                    </Box>
+                </Box>
+            </Box>
+
+        </>
+    );
 }
 
+
+
+
+// SUB-COMPONENT InstructionGrid
+
+const instructionStyles = makeStyles<Theme, InstructionProps> ({
+    headerText: {
+        width: '100%',
+        maxWidth: '1100px',
+        fontSize: '1.3rem',
+        textAlign: 'left',
+    },
+    gridBox: {
+        width: '100%',
+        overflow: 'hidden',
+        padding: '20px',
+        '& img': {
+            maxWidth: '100%',
+            height: 'auto',
+        },
+        '& h3': {
+            paddingBottom: '1em',
+        },
+    },
+});
+
+type InstructionProps = {
+    instructionList: { title: string, body: string, image: string }[],
+};
+
+function InstructionGrid(props: InstructionProps) {
+    const classes = instructionStyles(props);
+    var row = 0;
+
+    return (
+        <Grid container justify='space-between'>
+            {props.instructionList.map((instructionItem) => {
+                row++;
+                var text = (
+                    <Grid item sm={6} xs={12} className={`${classes.gridBox}`}>
+                        <Typography variant="h3" component="h3" align="left">
+                            {instructionItem.title}
+                        </Typography>
+                        <Typography variant="body1" component="div" align="left" className={`${classes.headerText}`}>
+                            {instructionItem.body}
+                        </Typography>
+                    </Grid>
+                );
+                var image = (
+                    <Grid item sm={6} xs={12} className={`${classes.gridBox}`}>
+                        <img src={instructionItem.image} alt="placeholder"></img>
+                    </Grid>
+                );
+                return (
+                    <>
+                        {/* Set order of the two jsx items - odd number rows have text first, even have image first */}
+                        {row % 2 == 0 ? image : text}
+                        {row % 2 == 0 ? text : image}
+                    </>
+                )
+            })}
+        </Grid>
+    );
+};
+
+
+
+// SUB-COMPONENT BulletGrid
+
+const bulletStyles = makeStyles<Theme, BulletProps> ({
+    headerText: {
+        width: '100%',
+        maxWidth: '1100px',
+        fontSize: '1.3rem',
+        textAlign: 'left',
+    },
+    grid: {
+        marginTop: '30px',
+    },
+    square: {
+        width: '100px',
+        height: '100px',
+        backgroundColor: 'white',
+        marginRight: '20px',
+    },
+});
+
+type BulletProps = {
+    list: string[],
+};
+
+function BulletGrid(props: BulletProps) {
```

Comment:
yeah i guess let's pop these into separate files, this one is getting lengthy
checkout the file structure of `AssetsView` from my branch in this PR https://github.com/Nonprofit-Exchange-Hub/web-app/pull/32/files

Reply:
Sure!